sideboard/lib/_websockets.py

```diff
@@ -143,7 +143,8 @@ def _refire_subscriptions(self):
         try:
             for cb in self._callbacks.values():
                 if 'client' in cb:
-                    self._send(method=cb['method'], params=cb['params'], client=cb['client'])
+                    params = cb['paramback']() if 'paramback' in cb else cb['params']
+                    self._send(method=cb['method'], params=params, client=cb['client'])
```

Comment:
This new feature is explained in the new documentation in ``index.rst`` above.  The tldr is that you can specify a function to be called anytime your websocket client connection disconnects and then reconnects.